rbd/rbd_options.go

```diff
@@ -0,0 +1,122 @@
+package rbd
```

Comment:
Can we please name the file "options.go"? The package dir is already rbd/ so having "rbd_" prefix the files seems a bit redundant to me.

Reply:
sure, I was just following the base filename `rbd.go`.